modules/src/ics04_channel/context.rs

```diff
@@ -0,0 +1,104 @@
+//! ICS4 (channel) context. The two traits `ConnectionReader` and `ConnectionKeeper` define
+//! the interface that any host chain must implement to be able to process any `ChannelMsg`.
+//! TODO make "ADR 004: IBC protocol implementation" for more details.
+//!
+//use crate::ics02_client::client_def::{AnyClientState, AnyConsensusState};
```

Comment:
The commented-out `use` statements in this file should probably be removed if you don't plan on uncommenting them later on.